conjur/host/host_controller.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+
+"""
+HostController module
+
+This module is the controller that facilitates all host actions
+"""
+
+# Builtins
+import sys
+
+class HostController():
+    """
+    HostController
+
+    This class represents the Presentation Layer for the User command.
+    """
+    def __init__(self, client, host_resource_data):
+        self.client=client
+        self.host_resource_data=host_resource_data
```

Comment:
for the refactoring part. should add input validation some way in the flow (host is not emty string and more).
maybe constructor of controllers is a good place to validate

Reply:
If it is empty i prompt them to enter in the host id. If they anyway enter a empty host, then it will fail. So that is some sort of validation no?

Reply:
yep.. you have it all covered.